mesonbuild/optinterpreter.py

```diff
@@ -72,10 +72,16 @@ def __init__(self, subproject, command_line_options):
         self.options = {}
         self.subproject = subproject
         self.sbprefix = subproject + ':'
-        self.cmd_line_options = {}
+        self.cmd_line_options = self.get_command_line_options(subproject,
+                                                              command_line_options)
+
+    @staticmethod
+    def get_command_line_options(subproject, command_line_options):
+        sbprfeix = subproject + ':'
```

Comment:
__[flake8]__
* [F841] local variable 'sbprfeix' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->